app/src/androidTest/java/org/mozilla/fenix/ui/MediaNotificationTest.kt

```diff
@@ -51,8 +51,10 @@ class MediaNotificationTest {
 
         navigationToolbar {
         }.enterURLAndEnterToBrowser(videoTestPage.url) {
+            verifyPageContent("Page content: video player")
```

Comment:
I think we can check against `videoTestPage.content.

Reply:
Ok, will  change that